.github/workflows/publish.yml

```diff
@@ -32,3 +32,7 @@ jobs:
       - uses: katyo/publish-crates@v2
         with:
           registry-token: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+  pypi-publish:
+    uses: tree-sitter/workflows/.github/workflows/package-pypi.yml@main
```

Comment:
do we trust `@main` or do you want to pin it to a hash?

Reply:
I think main is fine so that we benefit from any fixes. If it causes trouble we can always pin it later